app/lib/settings/pages/settings.dart

```diff
@@ -7,17 +7,48 @@ class SettingsPage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return ListView(children: [
-      Card(
-        child: ListTile(
-          leading: Icon(Icons.delete),
-          title: Text(context.l10n.settings_page_delete_data),
-          trailing: Icon(Icons.chevron_right),
-          onTap: () => showDialog(
-            context: context,
-            builder: (_) => _deleteDataDialog(context),
-          ),
+      ListTile(
+        title: Text(
+          context.l10n.settings_page_account_settings_headline,
+          style: PharmeTheme.textTheme.bodyLarge,
+        ),
+      ),
+      ListTile(
+        leading: Icon(Icons.delete),
+        title: Text(context.l10n.settings_page_delete_data),
+        trailing: Icon(Icons.chevron_right),
+        onTap: () => showDialog(
+          context: context,
+          builder: (_) => _deleteDataDialog(context),
+        ),
+      ),
+      Divider(),
+      ListTile(
+        title: Text(
+          context.l10n.settings_page_more_headline,
+          style: PharmeTheme.textTheme.bodyLarge,
         ),
       ),
+      ListTile(
+        title: Text(context.l10n.settings_page_onboarding),
+        trailing: Icon(Icons.chevron_right),
+        onTap: () => context.router.push(OnboardingRouter()),
+      ),
+      ListTile(
+        title: Text(context.l10n.settings_page_about_us),
+        trailing: Icon(Icons.chevron_right),
+        onTap: () => context.router.push(AboutUsRoute()),
+      ),
+      ListTile(
+        title: Text(context.l10n.settings_page_privacy_policy),
+        trailing: Icon(Icons.chevron_right),
+        onTap: () => context.router.push(PrivacyPolicyRoute()),
+      ),
+      ListTile(
+        title: Text(context.l10n.settings_page_terms_and_conditions),
+        trailing: Icon(Icons.chevron_right),
+        onTap: () => context.router.push(TermsAndConditionsRoute()),
+      ),
```

Comment:
We should either add icons to these `ListTile`s or remove the icon from the first `ListTile`. Currently, it looks kind of messy

Reply:
Any opinion on that @LeonHermann322? I agree that it should be consistent.

Reply:
Yes i agree, we should keep it consistent. Maybe remove the first one, but i don't have a strong opinion if you want to add icons to the later buttons.